orca_detector/Dockerfile.dev

```diff
@@ -25,17 +25,42 @@ RUN pip install --upgrade setuptools
 RUN pip3 install jupyter -U && \
     pip3 install jupyterlab
 
+# Clone the repo
+RUN git clone https://github.com/paloukari/OrcaDetector
+
 # Install all dependency libs for the project.
-COPY requirements.pip /
-RUN pip3 install -r ./requirements.pip
+RUN pip3 install -r OrcaDetector/orca_detector/requirements.pip
+
+# Download the data
+RUN apt-get install -y awscli
+
+ARG AWS_ID                                                                
+ARG AWS_SECRET                                                            
+RUN aws configure set aws_access_key_id ${AWS_ID}
+RUN aws configure set aws_secret_access_key ${AWS_SECRET}
+
+RUN aws s3 cp s3://w251-orca-detector-data/data.tar.gz .
+RUN tar -xvf data.tar.gz
+
+RUN aws s3 cp s3://w251-orca-detector-data/vggish_weights.tar.gz .
+RUN tar -xvf vggish_weights.tar.gz
+
+# Install and configure the SSH Server
```

Comment:
Yes, you should be able to keep the same experience.
..I'm more old school, I cannot do without a debugger :-)

Reply:
I think I'm the one that's developing "old school" -- debugging with `print()`.  Sometimes I even do quick edits in `vi` despite knowing of the existence of modern technology like debuggers :-)

Reply:
You are right :-)
It's funny how things change, people started coding like that because they didn't have the required tooling and today Data Scientists do the same driven by the DS tooling